libs/core/src/lib/popover/popover.component.ts

```diff
@@ -159,6 +166,16 @@ export class PopoverComponent extends BasePopoverClass implements AfterViewInit,
         if (this.popoverControl && this.triggers.includes('click')) {
             this.popoverControl.makeTabbable();
         }
+        this.listItems.forEach((li) => {
+            const listItem = this._getHtmlElement(li);
+            const listTitle = listItem.querySelector('.fd-list__title');
+            listTitle.setAttribute('tabindex', '-1');
+        });
+    }
```

Comment:
does not work with ContentChildren/ViewChildren as Mike suggested?

Reply:
It does uses the contentChildrens as the object to access the listItems, when adding the contentChildren to the directive(ListTitleDirective) the object is not created since there is not constructor defined in the directive, this throws a runtime exception. in order to avoid these i have used ListItemComponent and added the tab index.

<img width="779" alt="Screenshot 2021-08-09 at 9 14 06 AM" src="https://user-images.githubusercontent.com/53534379/128661737-6a03cfc3-053e-4dd4-91bd-06db30da52f3.png">

Reply:
> @Lokanathan-k @InnaAtanasova @fkolar this is possible via ContentChild and no need to hardcode the list title classname, see this commit: [684ac60](https://github.com/SAP/fundamental-ngx/commit/684ac60bff4c9194278756fe118d2e30ddd391b6)
> 
> However I think we should take it one step further and actually remove the tabIndex from the list title itself, if the list title is an anchor element, which I've done in this commit: [bc16508](https://github.com/SAP/fundamental-ngx/commit/bc1650885fcef546f54865a86791dd0a29e6c502)
> 
> If anyone objects we can roll back the most recent commit I've linked

I had this solution before i had this introduced my changes as @droshev had shared comments in my Previous Pr's not to introduce changes in different component when the issue is pointed to the Specific Component. So i found the best alternative solution without introducing the change in the ListComponent.

Thank you, @mikerodonnell89 @InnaAtanasova Helping resolving the issue.